.travis.yml

```diff
@@ -2,6 +2,12 @@ language: python
 sudo: false
 python:
     - 2.7
+    - 3.5
+    - 3.6
+matrix:
+    include:
+        - python: "3.7"
+          dist: xenial
```

Comment:
Travis CI is fun and exciting: the best practices change all the time!

This month's wisdom is:

- drop `sudo: false`
- move `dist: xenial` to the top level
- list all supported Python versions in the top-level `python:`
- (if you support pypy, you can (again) use `pypy` and `pypy3` to get the latest releases of pypy2.7 and pypy3.5)